src/Linters/AutoFixable.hack

```diff
@@ -0,0 +1,15 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST;
+
+<<__Sealed(AutoFixingLinter::class)>>
+interface AutoFixable {
```

Comment:
AutoFixingLinter takes a generic, which means `UnsafeBulkAutoFixes`
can't `require implements AutoFixingLinter<Terror>` without being generic itself.
It doesn't use the generic either, so this small marker interface is used instead.